File: app/src/main/java/cc/lixiaoyu/wanandroid/util/GlideImageLoader.java
package cc.lixiaoyu.wanandroid.util;

import android.content.Context;
import android.widget.ImageView;

import com.bumptech.glide.Glide;
import com.youth.banner.loader.ImageLoader;

import cc.lixiaoyu.wanandroid.entity.Banner;

public class GlideImageLoader extends ImageLoader {
    @Override
    public void displayImage(Context context, Object path, ImageView imageView) {
        Banner banner = (Banner) path;
        Glide.with(context).load(banner.getImagePath()).into(imageView);
    }
}

File: app/src/main/java/cc/lixiaoyu/wanandroid/util/GlideImageLoader.kt
package cc.lixiaoyu.wanandroid.util

import android.content.Context
import android.widget.ImageView
import cc.lixiaoyu.wanandroid.entity.Banner
import com.bumptech.glide.Glide
import com.youth.banner.loader.ImageLoader

class GlideImageLoader : ImageLoader() {
    override fun displayImage(context: Context, path: Any, imageView: ImageView) {
        val banner = path as? Banner ?: return
        Glide.with(context).load(banner.imagePath).into(imageView)
    }
}